dictionaries/zig/cspell-ext.json

```
@@ -0,0 +1,38 @@
+{
+    "id": "zig",
+    "version": "0.2",
+    "name": "Zig",
+    "description": "Zig dictionary for cspell.",
+    "readonly": true,
+    "dictionaryDefinitions": [
+        {
+            "name": "zig",
+            "path": "./dict/zig.txt",
+            "description": "Zig dictionary for cspell."
+        }
+    ],
+    "dictionaries": [],
+    "languageSettings": [
+        {
+            "languageId": "zig",
+            "locale": "*",
+            "includeRegExpList": [],
+            "ignoreRegExpList": [],
+            "patterns": [],
+            "dictionaries": ["zig"],
+            "dictionaryDefinitions": []
+        },
+        {
+            "languageId": "zon",
+            "locale": "*",
+            "ignoreRegExpList": ["/\\.hash =.*/"],
+            "dictionaries": ["zig"]
+        }
+    ],
+    "overrides": [
+        {
+            "filename": "**/*.zig.zon",
+            "languageId": "zon"
+        }
```

```suggestion
        {
            "filename": "**/*.zig.zon",
            "languageId": "zon"
        },
        {
            "filename": "**/*.zig",
            "languageId": "zig"
        }
```

CSpell doesn't currently know the `.zig` file extension.